### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1022] [Galaxy Trivia] [answered]
The number of Xs in each row and column should match the corresponding numbers for each row and column.

|  | 2 | 2 | 2 | 4 | 4 |
|---|---|---|---|---|---|
| **3** | | | | | |
| **2** | | | | | |
| **4** | | | | | | 
| **3** | | | | | | 
| **2** | | | | | |

**Comment by peterbaccaro** [comment, marked as answer]
![image](https://github.com/ParanoidUser/yolo/assets/74666079/e2aef1db-cf72-4e44-be93-5d8ca7fda735)
